test/passing/tests/extensions-indent.ml.ref

```diff
@@ -361,17 +361,45 @@ let _ = f [%ext new x]
 
 (* TODO: begin/end keywords must be preserved *)
 
-let _ = [%ext y >>= z]
+let _ =
+  [%ext
+       begin
+         y >>= z
+       end]
 
-let _ = [%ext y >>= z]
+let _ =
+  [%ext
+       begin
```

Comment:
This indentation looks like a bug. Should be 2 or 4 but not the column it broke.

Reply:
This indentation is due to the option `--extension-indent=5` for this file

Reply:
right!